scaffolding-ruby/lib/scaffolding.sh

```diff
@@ -393,7 +393,16 @@ scaffolding_create_dir_symlinks() {
 }
 
 scaffolding_create_files_symlinks() {
-  return 0
+  local entry source_file target
+
+  for entry in "${scaffolding_symlinked_files[@]}"; do
+    source_file="$scaffolding_app_prefix/$entry"
+    target="$pkg_svc_path/$entry"
```

Comment:
Right now this would link the file into the parent directory of `/hab/svc/<myapp>`--maybe we should add it under either the `var/` (`$pkg_svc_var_path`) or `files/` (`$pkg_svc_files_path`) directory so we don't pollute the service directory too much?